docs/contribute/docker-release-workflow.md

```diff
@@ -0,0 +1,54 @@
+# Docker Hub Release Workflow
+
+
+The workflow builds the **lind-wasm** Docker image from the *release* stage of `Dockerfile.e2e` and pushes it to Docker Hub as **`securesystemslab/lind-wasm`**.  
+It is **manual-only** (`workflow_dispatch`), so contributors must trigger it on demand.
+
+---
+
+## 1. Prerequisites
+
+| Requirement | Purpose |
+|-------------|---------|
+| Write rights on this repo | lets you add secrets & trigger the workflow |
+| Docker Hub access-token (or password) for `securesystemslab` | used to authenticate the image push |
```

Comment:
Could this be reworded to "used by GitHub Actions to authenticate the Docker Hub image push" or similar to clarify what authentication is being done?